common/plugins/org.polarsys.capella.common.helpers/src/org/polarsys/capella/common/helpers/cache/ModelCache.java

```diff
@@ -32,6 +33,7 @@ private ModelCache() {
    * @return If enabled, return the cached result if any or apply the function to the given parameter and cache the
    *         result before returning it.
    */
+  @Deprecated

```

Comment:
can you add a comment to explain why it's deprecated. 
(Use a CachedFunction instead to ensure that the cache is shared with all calls to this function rather than having a cache per calls )